runtime/bindings/python/tests/test_inference_engine/test_infer_request.py

```diff
@@ -187,3 +189,26 @@ def test_infer_mixed_keys(device):
     with pytest.raises(TypeError) as e:
         request.infer({0: tensor, "fc_out": tensor2})
     assert "incompatible function arguments!" in str(e.value)
+
+
+def test_infer_queue(device):
+    jobs = 8
+    num_request = 4
+    core = Core()
+    func = core.read_model(test_net_xml, test_net_bin)
+    exec_net = core.compile_model(func, device)
+    infer_queue = AsyncInferQueue(exec_net, num_request)
+    jobs_done = [{"finished": False, "latency": 0} for _ in range(jobs)]
+
+    def callback(request, job_id):
+        jobs_done[job_id]["finished"] = True
+        jobs_done[job_id]["latency"] = request.latency
```

Comment:
This works as `jobs_done` are "global" here. It can be done properly with passing references inside lambdas, however I do not know if this callback logic should be improved in temporary solution.

Reply:
Otherwise it looks good to me.